K번째수.js

```diff
@@ -0,0 +1,14 @@
+// arr.slice([begin[, end]])
+// Array.map // map() 메서드는 배열 내의 모든 요소 각각에 대하여 주어진 함수를 호출한 결과를 모아 새로운 배열을 반환합니다.
+function solution(array, commands) {
+    let answer = [];
```

Comment:
```suggestion
    let answer = [];
```
![image](https://github.com/hobiJeong/Jbhcdstory/assets/83463918/637d12c3-1dbb-45c6-8805-5461e00238fb)

https://nenara.com/75